src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -872,6 +876,25 @@ export default class DefaultAudioVideoController
     return true;
   }
 
+  updateLocalVideoFromPolicy(): boolean {
+    if (this.mayNeedRenegotiationForSimulcastLayerChange) {
+      this.logger.info('Needs regenotiation for local video simulcast layer change');
+      this.mayNeedRenegotiationForSimulcastLayerChange = false;
+      return false;
+    }
+
+    // Update bandwidth without renegotiation
+    this.logger.info('Updating local video from policy without renegotiation');
+    if (this.meetingSessionContext.enableSimulcast) {
+      // AttachMediaInputTask will update simulcast streams encoding change
```

Comment:
Nit: Wrap symbols in `.  Also can you be more explicit ? like are we updating the tranceivers here?